keyboards/jj_ergo/keymaps/default/default.png.md

```diff
@@ -0,0 +1 @@
+https://i.imgur.com/fKX0Zbs.png
```

Comment:
Not sure what the idea is with these markdown files - looks like you want it to display the image linked but GitHub just shows the link for me.

Please rename this file to `readme.md`, delete the other one and apply the markdown image syntax:
```suggestion
![](https://i.imgur.com/fKX0Zbs.png)
```

But I would also prefer to see this more in text format instead, for accessibility.

Reply:
@fauxpark These are actually copied from the Ergodox EZ default keymap, and should possibly be changed there as well.

Also I'm not sure how ZSA would feel about their images being used in this way. @drashna should probably advise.

Reply:
Personally, I don't have an issue with this, but I've passed the question along, just in case.

Reply:
It's all good, use the images! :) Go right ahead.